.vib/suitecrm/goss/suitecrm.yaml

```diff
@@ -0,0 +1,39 @@
+command:
+  check-app-version:
+    exec: grep '"version":' /opt/bitnami/suitecrm/package.json
+    exit-status: 0
+    stdout:
+      - '"version": "{{ .Env.APP_VERSION }}",'
+  check-enabled-modules:
+    exec: php -m
+    exit-status: 0
+    stdout:
+    {{ range $module := .Vars.phpmodules }}
+      - "{{ $module }}"
+    {{ end }}
+file:
```

Comment:
Looking at our compilation logic, we could also add a check to ensure we are not including the `/opt/bitnami/suitecrm/node_modules` directory, wdyt?

Reply:
Good point! Will add it